apps/challenges/views.py

```diff
@@ -816,6 +817,28 @@ def create_challenge_using_zip_file(request, challenge_host_team_pk):
             response_data = {"error": message}
             return Response(response_data, status=status.HTTP_400_BAD_REQUEST)
 
+        # To ensure that the schema for submission metaattributes is valid.
```

Comment:
`# To ensure that the schema for submission metaattributes is valid.` -> `# To ensure that the schema for submission meta attributes is valid.`